### ISSUE TITLE
datetime picker - start/end date

### ISSUE THREAD

**Issue by smensel** [alfonsobries/vue-tailwind#194]
Hello,
is it possible to set a start and/or end date for the datetime picker? 
E.g. I would like to show as selection only the last 10 years until today.

**Comment by alfonsobries** [comment]
@smensel yes you can use a function to disable any date that is < 10 years see https://www.vue-tailwind.com/docs/datepicker#disable-dates
